Updates to install/configuration (#37)
AdobeDocs/commerce-extensibility: src/pages/events/project-setup.md

*  Have an Adobe Developer account with System Administrator or Developer Role permissions. [Getting started with Adobe Developer Console](https://developer.adobe.com/developer-console/docs/guides/getting-started/) describes how to enroll in the Adobe developer program.

*  Be familiar with [Adobe I/O Runtime](https://developer.adobe.com/runtime/docs/guides/getting-started/).

*  Install the [`aio CLI`](https://developer.adobe.com/runtime/docs/guides/getting-started/setup/)

*  Have access to an Adobe Commerce on cloud infrastructure or to an on-premises instance.

## Set up a project

[Projects Overview](https://developer.adobe.com/developer-console/docs/guides/projects/) describes the different types of projects and how to manage them. Here, we'll create a templated project.

1. Log in to the Adobe Developer Console and select the desired organization from the dropdown menu in the top-right corner.

1. Click **Create new project** > **Project from template**.

   ![Create a project](../_images/create-project.png)

1. Select **App Builder**. The **Set up templated project** page displays.

   ![Add an api](../_images/set-up-templated-project.png)

> 1. Specify a project title and app name. Make sure the **Include Runtime with each workspace** check box is selected. Click **Save**. The Console creates a Production and a Stage workspace.

Reviewer: ```suggestion
1. Specify a project title and app name. Make sure the **Include Runtime with each workspace** checkbox is selected. Click **Save**. The Console creates a Production and a Stage workspace.
```

   ![Add an api](../_images/workspaces.png)

>    Each App Builder project has two default workspaces: Production and Stage. You can add more workspaces as needed. In this procedure, we'll set up the Stage workspace.

Reviewer: ```suggestion
   Each App Builder project has two default workspaces: Production and Stage. You can add more workspaces as needed. In this procedure, we will set up the Stage workspace.
```

1. Click **Save configured API**.

1. Unzip the downloaded `config.zip` file. The extracted `config` directory should contain a `certificate_pub.crt` and a `private.key` file. The `private.key` file is required to configure the Commerce Admin.

## Download the workspace configuration file

The console can generate a JSON file that defines the configuration of your workspace. You will use this file to configure the Commerce Admin.

To download a `.json` file containing your workspace configuration:

1. Go the overview page of your workspace (Staging).

> 1. Click the **Download All** button at the top right corner.

Reviewer: ```suggestion
1. Click the **Download All** button in the top-right corner.
```

   ![download the workspace config](../_images/download-workspace-config.png)

   The `<Workspace-name>.json` file downloads automatically. In this example, the file is named `485PeachHare-283976-Stage.json`.

## Set up App Builder and define a runtime action

The first step to setting up your App Builder template is to set up your environment and create a runtime action. For details about this process, see [Setting up Your Environment](https://developer.adobe.com/runtime/docs/guides/getting-started/setup/).

1. Create a project directory on your local filesystem and change to that directory.

   ```bash
   mkdir myproject && cd myproject
   ```

1. Log in to Adobe IO from a terminal:

   ```bash
   aio login
   ```

   Your web browser displays the login page.

1. Enter your Adobe ID credentials.

> 1. Close the browser tab and return to your terminal. Enter the following command to bootstrap your application.

Reviewer: ```suggestion
1. Close the browser tab and return to your terminal. Enter the following command to bootstrap your application:
```

   ```bash

>    aio app init   

Reviewer: ```suggestion
   aio app init
```
extra space
